Tests/ClientRuntimeTests/NetworkingTests/URLSession/FoundationStreamBridgeTests.swift

```diff
@@ -5,13 +5,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-// Run this unit test on macOS only.
-//
-// `FoundationStreamBridge` is not usable on Linux because it uses ObjC-interop features.
-//
-// Unit tests of types that spawn threads are unreliable on Apple-platform simulators.
-// Mac tests "run on metal", not a simulator, so they will run reliably.
-#if os(macOS)
+// `FoundationStreamBridge` is not usable on Linux because it uses ObjC-interop features,
+// so this test is disabled there.
+
+#if os(iOS) || os(macOS) || os(watchOS) || os(tvOS) || os(visionOS)
```

Comment:
Now that we don't use a Stream & RunLoop in `FoundationStreamBridge`, we can test it on all the Apple platforms & their simulators.